.stylelintrc.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I never know if config in hidden files or not is preferable, but I guess this makes sense since we have `.eslintrc`